packages/calcite-components/src/components/modal/modal.tsx

```diff
@@ -343,6 +342,15 @@ export class Modal
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * We use an internal property to handle styles for when a modal is actually opened, not just when the open attribute is applied. This is a property because we need to apply styles to the host element and to keep the styles present while beforeClose is .
```

Comment:
Nitpicks: 

* `...while beforeClose is .` ➡️  `...while beforeClose is.`
* `@internal.` ➡️ `@internal`